lib/identity/stateTransitions/identityCreateTransition/applyIdentityCreateTransitionFactory.js

```diff
@@ -20,8 +20,8 @@ function applyIdentityCreateTransitionFactory(
    * @return {Promise<void>}
    */
   async function applyIdentityCreateTransition(stateTransition) {
-    const output = stateTransition.getAssetLock().getOutput();
-    const outPoint = stateTransition.getAssetLock().getOutPoint();
+    const output = stateTransition.getAssetLock().getProof().getOutput();
+    const outPoint = stateTransition.getAssetLock().getProof().getOutPoint();
 
     const creditsAmount = convertSatoshiToCredits(output.satoshis);
 
```

Comment:
We could rename `StateRepository#storeAssetLockTransactionOutPoint` to `StateRepository#markAssetLockTransactionOutPointAsUsed` to make it consistent with `StateRepository#isAssetLockTransactionOutPointAlreadyUsed`